CHANGELOG.md

```
@@ -2,6 +2,7 @@
 
 #### 9.5.7 - SNAPSHOT
   * Enhanced the MX parser log verbosity when parsing malformed content
+  * Added new logic to MxNode to removing all leaves with empty attributes and empty content
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)